gammapy/maps/region/geom.py

```diff
@@ -794,6 +793,7 @@ def plot_region(self, ax=None, kwargs_point=None, path_effect=None, **kwargs):
         ax : `~astropy.visualization.WCSAxes`
             Axes to plot on.
         """
+        from gammapy.visualization.utils import ARTIST_TO_LINE_PROPERTIES
```

Comment:
I thunk this can remain a non-delayed import...

Reply:
Done